### ISSUE TITLE
Bad metrics keys for glyphs with stylistic alternate feature suffixes

### ISSUE THREAD

**Issue by stephenhutchings** [jayKayEss/MetricsSolver#5]
Glyphs with feature code suffixes (eg. `a.ss01`) fail to have their metrics solved.

```
WARNING: Bad metrics key a.ss found in glyph u.ss01
```

I assume this is because the [matcher regex](https://github.com/jayKayEss/MetricsSolver/blob/720b109a4436b0799cf2127c4321a8d2380f88a8/MetricsSolver.glyphsPlugin/Contents/Resources/matcher.py#L5) isn't looking for digits. Happy to make a pull request if that helps.

**Comment by albertomalossi** [comment]
I think it does solve them, it just reports them as bad in the macro window.

**Comment by lowy-institute-digital-team** [comment]
@albertomalossi You're right, checking again the plugin does solve them as expected. Ideally though, the regex would still be tweaked to allow digits. Or are these metrics keys actually throwing an error for a good reason?

**Comment by stephenhutchings** [comment]
@albertomalossi You're right, checking again the plugin does solve them as expected. Ideally though, the regex would still be tweaked to allow digits. Or are these metrics keys actually throwing an error for a good reason?

**Comment by albertomalossi** [comment]
@stephenhutchings I totally agree, the regex shouldn't throw an error when it detects digits, bad metrics or not, especially when it solves them anyway. Unfortunately the plugin seems semi-abandoned.
